src/svg.rs

```diff
@@ -2,19 +2,21 @@
 // See the COPYRIGHT file at the top-level directory of this distribution.
 // Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
 
+#[cfg(any(all(feature = "svg", feature = "v1_16"), feature = "dox"))]
+use enums::SvgUnit;
+use enums::{Status, SurfaceType, SvgVersion};
+use ffi;
 use std::convert::TryFrom;
 use std::ffi::{CStr, CString};
 use std::fmt;
 use std::io;
 use std::mem;
 use std::ops::Deref;
+#[cfg(not(windows))]
+use std::os::unix::prelude::*;
```

Comment:
Without a windows case this is not going to compile on Windows. Looks ok for UNIX though.